.travis.yml

```diff
@@ -1,21 +1,43 @@
 language: php
+
+sudo: false
+
 php:
-  - "5.5"
-  - "5.4"
-  - "5.3"
+  - 7.0
+  - 5.6
+  - 5.5
+  - 5.4
+  - 5.3
+  - hhvm
+
+matrix:
+  fast_finish: true
+  include:
+    - php: 5.6
+      env: ES_VERSION="1.6.2"
+    - php: 5.6
+      env: ES_VERSION="1.5.2"
+    - php: 5.6
+      env: ES_VERSION="1.4.5"
+    - php: 5.6
+      env: ES_VERSION="1.3.9"
+
 env:
-  - ES_VERSION=0.90.0 ES_TEST_HOST=http://localhost:9200
-  - ES_VERSION=0.90.1 ES_TEST_HOST=http://localhost:9200
-  - ES_VERSION=0.90.2 ES_TEST_HOST=http://localhost:9200
-  - ES_VERSION=0.90.3 ES_TEST_HOST=http://localhost:9200
-  - ES_VERSION=0.90.4 ES_TEST_HOST=http://localhost:9200
-  - ES_VERSION=0.90.5 ES_TEST_HOST=http://localhost:9200
+  global:
+    - ES_VERSION="1.7.1"
+    - ES_TEST_HOST=http://localhost:9200
+
+before_install:
+  - ./travis/download_run_es.sh
```

Comment:
@TomasVotruba The file seems to be called ./travis/download_and_run_es.sh (the and is missing). That is why all travis builds already fail at the moment.